python/tvm/auto_scheduler/measure.py

```diff
@@ -303,11 +306,16 @@ class LocalBuilder(ProgramBuilder):
         This is used in a wrapper of the multiprocessing.Process.join().
     n_parallel : int = multiprocessing.cpu_count()
         Number of threads used to build in parallel.
-    build_func : str = 'default'
-        The name of registered build function.
+    build_func: callable or str
```

Comment:
add the default value in the comment

Reply:
I have added BuildFunc default name = "default" and default func callable = tar.tar